src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -534,7 +535,7 @@ s32 EnSkb_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
         } else {
             *dList = NULL;
         }
-    } else if ((limbIndex == 12) && ((this->unk_283 & 2) != 0)) {
+    } else if ((limbIndex == 12) && ((this->bodybreakState & 2) != 0)) { // jaw, don't draw if headless
```

Comment:
I'm not sure about the name for `bodybreakState`,

1) it's flags, not a single state value
2) it's not just for the BodyBreak system, it's also for the actor's "own" usage

I don't have a suggestion right now

Reply:
Yeah this is one of the things I am least happy about... it's usage is very weird to me, but certainly related to bodybreak stuff. I wanted to give names to the values they use, but I honestly couldn't come up with any sensible naming scheme.

0 is "nothing special"
1 is "prepare for becoming headless"
2 is "currently headless"
4 is "prepare for falling apart completely"
8 is "done falling apart"

as best I can tell.

Reply:
after most recent review, going with `breakFlags`